package.json

```diff
@@ -40,19 +40,17 @@
     "@0x/types-v2": "npm:@0x/types@2.4.3",
     "@0x/utils": "^5.1.2",
     "@airswap/order-utils": "^0.3.20",
-    "@babel/core": "^7.1.6",
-    "@babel/preset-env": "^7.1.6",
-    "babel-jest": "^24.9.0",
     "ganache-core": "^2.11.0-beta.0",
-    "glob": "^7.1.3",
-    "jest": "^24.1.0",
-    "jest-matcher-utils": "^24.9.0",
+    "jest": "^26.1.0",
+    "jest-matcher-utils": "^26.1.0",
     "request-promise": "^4.2.2",
-    "solhint": "^2.3.0",
+    "solhint": "^3.0.0",
     "truffle": "^5.1.32",
-    "web3": "1.2.4",
-    "web3-eth-abi": "1.2.4",
-    "web3-utils": "1.2.4"
+    "ts-jest": "^26.1.1",
+    "typescript": "^3.9.5",
```

Comment:
I see you've found a way to sneak typescript back in here :P 

Is this purely to do the work that babel packages were doing, or are we using TS for something else now?

Reply:
This is a replacement for babel, yes and to make the path aliasing (~/) work with IDEs autocompletion for paths, etc. Currently no further TS features (no typechecking, etc.). As mentioned before, I'd be open and willing to implement that though (and still think it would be beneficial).